Projects/Server/World/Map.cs

```diff
@@ -384,6 +418,23 @@ public string Name
 
         public static int MaxLOSDistance { get; set; } = 25;
 
+        public static void RegisterMap(MapDefinition mapDefinition)
```

Comment:
I am seeing an infinite recursion. Might want to do aggressive inlining and see if that fixes it?